.pre-commit-config.yaml

```diff
@@ -15,21 +15,21 @@ repos:
     rev: v1.71.0
     hooks:
       - id: terraform_fmt
-      - id: terraform_tflint
-        args:
-          - "--args=--only=terraform_deprecated_interpolation"
-          - "--args=--only=terraform_deprecated_index"
-          - "--args=--only=terraform_unused_declarations"
-          - "--args=--only=terraform_comment_syntax"
-          - "--args=--only=terraform_documented_outputs"
-          - "--args=--only=terraform_documented_variables"
-          - "--args=--only=terraform_typed_variables"
-          - "--args=--only=terraform_module_pinned_source"
-          - "--args=--only=terraform_naming_convention"
-          - "--args=--only=terraform_required_version"
-          - "--args=--only=terraform_required_providers"
-          - "--args=--only=terraform_standard_module_structure"
-          - "--args=--only=terraform_workspace_remote"
+      # - id: terraform_tflint
+      #   args:
+      #     - "--args=--only=terraform_deprecated_interpolation"
+      #     - "--args=--only=terraform_deprecated_index"
+      #     - "--args=--only=terraform_unused_declarations"
+      #     - "--args=--only=terraform_comment_syntax"
+      #     - "--args=--only=terraform_documented_outputs"
+      #     - "--args=--only=terraform_documented_variables"
+      #     - "--args=--only=terraform_typed_variables"
+      #     - "--args=--only=terraform_module_pinned_source"
+      #     - "--args=--only=terraform_naming_convention"
+      #     - "--args=--only=terraform_required_version"
+      #     - "--args=--only=terraform_required_providers"
+      #     - "--args=--only=terraform_standard_module_structure"
+      #     - "--args=--only=terraform_workspace_remote"
```

Comment:
why is this commented out? it would be really nice if we can have tflint as part of precommit for both azure and aws

Reply:
Was slowing me down during the test and forgot to turn it back. Thanks for pointing out